gui/mozregui/wizard.py

```diff
@@ -185,9 +186,21 @@ class InboundPage(WizardSelectionRangePage):
     SUBTITLE = "Select the inbound changesets range."
     FIELDS = {"start_changeset": "start_changeset",
               "end_changeset": "end_changeset",
+              "build_type": "build_type",
               "inbound_branch": "inbound_branch"}
     ID = 2
 
+    def __init__(self):
+        WizardPage.__init__(self)
+        self.build_type_model = QStringListModel()
+        self.ui.build_type.setModel(self.build_type_model)
+
+    def initializePage(self):
+        WizardSelectionRangePage.initializePage(self)
+        application_name = self.field('application').toPyObject()
+        self.build_type_model = QStringListModel(
+            [FC_REGISTRY.get(str(application_name)).BUILD_TYPES])
+
```

Comment:
@parkouss  I'm curious about why it keeps telling me  no attribute 'BUILD_TYPES'

Traceback (most recent call last):
  File "/Users/mikeling/mozregression/gui/mozregui/wizard.py", line 202, in initializePage
    [FC_REGISTRY.get(str(application_name)).BUILD_TYPES])
AttributeError: type object 'FirefoxConfig' has no attribute 'BUILD_TYPES'